aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1559,13 +1559,13 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
                     deleteInlineStyleFromTheBeginning()
                 }
             }
-            // Fix for crash when pasting text on Samsung Devices running Android 8.
+            // Fix for crash when pasting text on Samsung Devices running Android 8 & 7.
             // Ref: https://github.com/wordpress-mobile/WordPress-Android/issues/8827
```

Comment:
We might want to consider adding a link to the new issue - https://github.com/wordpress-mobile/WordPress-Android/issues/10872. Wdyt?

Reply:
Yes, that's a good point. Will do!

Reply:
Resolved in af1e734 Let me know if this works. I was gonna convert this to a multiline comment but didn't see any other usages in this manner outside of for methods so just left it with the same format.